modules/tinymce/src/core/test/ts/browser/commands/ContentLangTest.ts

```diff
@@ -0,0 +1,84 @@
+import { beforeEach, context, describe, it } from '@ephox/bedrock-client';
+import { TinyAssertions, TinyHooks, TinySelections } from '@ephox/mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+import Theme from 'tinymce/themes/silver/Theme';
+
+describe('browser.tinymce.core.fmt.ContentLangTest', () => {
+  const hook = TinyHooks.bddSetupLight<Editor>({
+    base_url: '/project/tinymce/js/tinymce'
+  }, [ Theme ]);
+
+  context('lang format plays nicely with other formats', () => {
+    beforeEach(() => {
+      const editor = hook.editor();
+      editor.setContent('<p>Hello world</p>');
+      TinySelections.setSelection(editor, [ 0, 0 ], 0, [ 0, 0 ], 'Hello world'.length);
+    });
+
+    it('TINY-6149: supports setting language then setting color', () => {
```

Comment:
Could be misunderstanding but should the labels for this test and the below test be reversed?

Reply:
Good catch